system/Filters/Filters.php

```diff
@@ -240,6 +240,16 @@ public function initialize(string $uri = null)
 		$this->processMethods();
 		$this->processFilters($uri);
 
+		// Set the toolbar filter to the last position to be executed
+		if (in_array('toolbar', $this->filters['after']) &&
```

Comment:
can use strict in array: 

```suggestion
              if (in_array('toolbar', $this->filters['after'], true) &&
```

Reply:
Thanks!